learn/wavemaker-release-notes/v10-10-0.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "WaveMaker 10.10.0 - Release date: 22 November 2021"
+id: ""
+sidebar_label: "Release v10.10.0"
+---
+*WaveMaker released a new update version with some new features and bug fixes.*
+
+## New Features
+
+---
+
+### Prefab Localization
+
+#### Override Prefab locale messages in the application
+
+You can override the values for the Prefab local keys by using the "prefabMessages" object in the I18N folder. 
+
+- For example, a Prefab contains languages including English, Spanish, and Arabic. To edit a prefab message key of Spanish, you override the message keys in the `es.json` file where the prefab messages object is provided, placed in the I18N folder. Similarly, to edit Arabic, you edit the message keys in the `ar.json` file.
+
+#### Handling unsupported Prefab locale
+
+- Suppose the application supports a locale that Prefab does not support. In that case, the prefab messages are picked from the Prefabs' en.json file for that specific locale in the output folder instead of empty messages. 
```

Comment:
Also mention, "Even though the locale is not supported in the prefab, you can still provide localized messages in that language by extending the language.json file and adding keys manually." E.g. if prefab supports only English and Spanish but app supports English, Spanish and Arabic, you can add the respective locale message keys in the prefabMessages object inside the ar.json file.